src/librustc_mir/transform/mod.rs

```diff
@@ -185,6 +185,41 @@ pub fn run_passes(
     body.phase = mir_phase;
 }
 
+fn mir_const_qualif(tcx: TyCtxt<'_>, def_id: DefId) -> u8 {
```

Comment:
A doc comment would be good especially for the return type.

Reply:
I think we can replace it now with:
```rust
struct ConstQualif {
    has_mut_interior: bool,
    needs_drop: bool,
}
```

Reply:
I think we can replace it now with:
```rust
struct ConstQualif {
    has_mut_interior: bool,
    needs_drop: bool,
}
```

Reply:
This change was made in the latest version.